pyalgostrategypool/options_bear_call_ladder_v2.py

```diff
@@ -0,0 +1,91 @@
+        self.instruments_done_for_the_day = None
+
+    def initialize(self):
+        super().initialize()
+        self.instruments_done_for_the_day = []
+
+    def get_child_instrument_details(self, base_instrument, tradingsymbol_suffix, strike_direction, no_of_strikes):
+        expiry_date = self.get_allowed_expiry_dates()[0]
+        child_instrument = self.get_options_instrument_with_strike_direction(base_instrument, expiry_date, tradingsymbol_suffix, strike_direction, no_of_strikes)
+        return child_instrument
+
+    def options_instruments_set_up_local(self, base_instrument, tradingsymbol_suffix, current_close, direction=None):
+        self.logger.info(f"delete this : {base_instrument} {tradingsymbol_suffix} {current_close} {direction}")
+        expiry_dates = self.get_allowed_expiry_dates()
+        for expiry_date in expiry_dates:
+            self.options_instruments_set_up(base_instrument, direction, expiry_date, tradingsymbol_suffix, current_close)
+
+    def strategy_select_instruments_for_entry(self, candle, instruments_bucket):
+        selected_instruments, meta = [], []
+
+        for instrument in instruments_bucket:
+
+            # Process entry signal(s) only if instrument is not processed already
+            if instrument not in self.instruments_done_for_the_day:
+                self.instruments_done_for_the_day.append(instrument)
+                # get ltp of base instrument
+                ltp = self.broker.get_ltp(instrument)
+
+                # setup child instruments
+                self.options_instruments_set_up_local(instrument, "CE", ltp)
+                self.options_instruments_set_up_local(instrument, "PE", ltp)
+
+                leg_wise_list = [('LEG_ONE', self._leg_one_tradingsymbol_suffix, self._leg_one_strike_direction, self._leg_one_number_of_strikes, self._leg_one_transaction_type),
+                                 ('LEG_TWO', self._leg_two_tradingsymbol_suffix, self._leg_two_strike_direction, self._leg_two_number_of_strikes, self._leg_two_transaction_type),
+                                 ('LEG_THREE', self._leg_three_tradingsymbol_suffix, self._leg_three_strike_direction, self._leg_three_number_of_strikes, self._leg_three_transaction_type)]
+
+                for leg_number, tradingingsymbol_suffix, strike_direction, number_of_strikes, transaction_type in leg_wise_list:
+                    self.logger.info(f'Processing {leg_number}...')
+                    child_instrument = self.get_child_instrument_details(instrument, self.tradingsymbol_suffix_map[tradingingsymbol_suffix], self.strike_direction_map[strike_direction], number_of_strikes)
+                    selected_instruments.append(child_instrument)
+                    meta.append({'base_instrument': instrument, 'action': self.transaction_type_map[transaction_type]})
+
+        return selected_instruments, meta
+
+    def strategy_enter_position(self, candle, instrument, sideband_info):
+        # Place buy order
```

Comment:
`# Place buy order`
remove comment

Reply:
search for this in all strategies, and remove

Reply:
done